dpdispatcher/submission.py

```diff
@@ -1191,6 +1198,7 @@ def arginfo(detail_kwargs=True):
             ),
             Argument("queue_name", str, optional=True, doc=doc_queue_name, default=""),
             Argument("group_size", int, optional=False, doc=doc_group_size),
+            Argument("sge_pe_name", str, optional=True, doc=doc_sge_pe_name),
```

Comment:
This is a `Machine` specific argument and should not be added to the common arguments. Instead, add to `kwargs` in `Machine.resources_subfields`.

https://github.com/deepmodeling/dpdispatcher/blob/ec6081be706a50f647a2ded18a1f8c8f429bdbea/dpdispatcher/machines/lsf.py#L160-L216